### PR TITLE
Update product.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Removed redundant content and added headings to edition information as per: https://github.com/mattermost/docs/issues/1590.

### CODE REVIEW

File: source/overview/product.rst [hunk 1]
```diff
@@ -24,31 +24,30 @@ Mattermost Editions
...
-**Mattermost Enterprise Edition E10** is a commercial enterprise messaging solution for teams, groups and departments working on multiple projects scaling from hundreds to thousands of users. E10 is your "virtual campus", offering security, configuration and access control options for larger organizations, including:
+Mattermost Enterprise Edition E10
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A commercial enterprise messaging solution for teams, groups and departments working on multiple projects scaling from hundreds to thousands of users. E10 is your "virtual campus", offering security, configuration and access control options for larger organizations, including:
 
-- Active Directory / LDAP single-sign-on
+- Active Directory/LDAP single-sign-on
 - Encrypted push notifications with service level agreements (SLAs) via HPNS
 - Multi-factor authentication
```
Comment (jasonblais) [inline source/overview/product.rst]: Multi-factor authentication is now in fact a Team Edition feature

File: source/overview/product.rst [hunk 2]
```diff
@@ -24,31 +24,30 @@ Mattermost Editions
...
-**Mattermost Team Edition** - A free-to-use, open source, self-hosted alternative to proprietary SaaS messaging. Team Edition is your open source "virtual office", offering all the core productivity benefits of competing SaaS solutions. It deploys as single Linux binary with MySQL or PostgreSQL under an MIT license. Features include:
-
-- One-to-one and group messaging, file sharing and unlimited search history
+- One-to-one and group messaging, file sharing, and unlimited search history
 - Native apps for iOS, Android, Windows, Mac, Linux
-- Threaded messaging, emoji and custom emoji
+- Threaded messaging, emoji, and custom emoji
 - Tools for custom branding
-- Highly customizable `third party bots, integrations <https://about.mattermost.com/community-applications/#publicApps>`__ and `command line tools <https://docs.mattermost.com/administration/command-line-tools.html>`__
+- Continuous archiving and unlimited search
```
Comment (jasonblais) [inline source/overview/product.rst]: `unlimited search history` is mentioned in the first bullet - not sure if this is a duplicate or if you have another use case in mind?

File: source/overview/product.rst [hunk 3]
```diff
@@ -24,31 +24,30 @@ Mattermost Editions
...
+- Highly customizable `third party bots, integrations <https://about.mattermost.com/community-applications/#publicApps>`__, and `command line tools <https://docs.mattermost.com/administration/command-line-tools.html>`__
 - Extensive integration support via `webhooks, APIs, drivers <https://docs.mattermost.com/guides/integration.html>`__ and `third party extensions <https://about.mattermost.com/default-app-directory/>`__
 - Easily scalable from dozens to hundreds of users
 - New improvements released every two months
 - Languages include U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Russian, Turkish, Spanish, and Ukrainian.
```
Comment (jasonblais) [inline source/overview/product.rst]:
```suggestion
 - Languages include U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, and Ukrainian.
```

File: source/overview/product.rst [hunk 4]
```diff
@@ -57,7 +56,9 @@ To get started, `download the open source Mattermost Team Edition server <https:
...
 
-**Mattermost Enterprise Edition E20** is commercial enterprise-grade messaging system that scales to hundreds to tens of thousands of users. E20 is your "virtual city", with cluster-based horizontal scaling, high availability deployment, performance monitoring systems and compliance reporting plus advanced corporate directory integration. It offers the following advanced features **in addition to** E10:
+Mattermost Enterprise Edition E20
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A commercial enterprise-grade messaging system that scales from hundreds to tens of thousands of users. E20 is your "virtual city", with cluster-based horizontal scaling, high availability deployment, performance monitoring systems and compliance reporting plus advanced corporate directory integration. Building on the features in E10, it offers the following advanced features:
 
 - Advanced SAML 2.0 authentication with Okta, OneLogin, and Active Directory Federation Services
```
Comment (jasonblais) [inline source/overview/product.rst]: I would add the following to the list 

```
 - Custom retention policies for messages and files
 - Active Directory / LDAP Group sync
 - Elasticsearch support for highly efficient database searches in a cluster environment
```

Retention policies above or below compliance exports; LDAP group sync below SAML; elasticsearch below horizontal scaling.

**Comment by jasonblais** [comment]
Since this is for the Product Overview page, also requesting review from @lfbrock as Head of Product (both as FYI and if she has any suggestions)

File: source/overview/product.rst (hunk 2 above)
Comment (justinegeffen) [reply source/overview/product.rst]: No; oversight on my part. Thanks for picking that up!

File: source/overview/product.rst [hunk 5]
```diff
@@ -22,47 +22,50 @@ Core committers, including both community contributors and paid staff at Matterm
 Mattermost Editions
 -----------------------
 
-Mattermost is offered in three editions:
+Mattermost is offered in three editions, in multiple languages including U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, and Ukrainian.
```
Comment (justinegeffen) [inline source/overview/product.rst]: I think this fits better here. In a similar vein, I propose adding MFA to this introduction as it's applied across all versions. 3/5.

File: source/overview/product.rst [hunk 6]
```diff
@@ -22,47 +22,50 @@ Core committers, including both community contributors and paid staff at Matterm
...
-- Languages include U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Russian, Turkish, Spanish, and Ukrainian.
+- New features and improvements released every two months
 
 To get started, `download the open source Mattermost Team Edition server <https://about.mattermost.com/download>`__ under an MIT license.
 
-**Mattermost Enterprise Edition E10** is a commercial enterprise messaging solution for teams, groups and departments working on multiple projects scaling from hundreds to thousands of users. E10 is your "virtual campus", offering security, configuration and access control options for larger organizations, including:
+Mattermost Enterprise Edition E10
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A commercial enterprise messaging solution for teams, groups and departments working on multiple projects scaling from hundreds to thousands of users. E10 is your "virtual campus", offering security, configuration and access control options for larger organizations, including:
```
Comment (justinegeffen) [inline source/overview/product.rst]: Should we include something like "building on the features offered in Team Edition"? We have a similar line in the E20 description and wondering if we either include that here as well or remove it from both descriptions? If we remove it then a link to here: https://mattermost.com/pricing-feature-comparison/ at the start of the section would work.

File: source/overview/product.rst (hunk 5 above)
Comment (jasonblais) [reply source/overview/product.rst]: @justinegeffen All of the Team Edition features apply across all versions. Perhaps we should make that more clear -- e.g. for E10 we say `Building on the features in Team Edition, it offers the following advanced features:`

File: source/overview/product.rst (hunk 6 above)
Comment (jasonblais) [reply source/overview/product.rst]: Yes I think we can -- but we could also link to the feature comparison page in the intro (I'm 0/5)

File: source/overview/product.rst (hunk 5 above)
Comment (justinegeffen) [reply source/overview/product.rst]: Agreed. I've given my MFA suggestion more thought. Adding your suggestion would accomplish the same goal as my more vague one so I'm going to go with that.

Comment (justinegeffen) [reply source/overview/product.rst]: I've added the link to the final paragraph of the document.

File: source/overview/product.rst [hunk 7]
```diff
@@ -22,50 +22,53 @@ Core committers, including both community contributors and paid staff at Matterm
 Mattermost Editions
 -----------------------
 
-Mattermost is offered in three editions:
+Mattermost is offered in three editions, in multiple languages including U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, and Ukrainian.
```
Comment (jasonblais) [inline source/overview/product.rst]: @justinegeffen I'm curious why you feel it fits better here vs as part of Team Edition features?

Comment (justinegeffen) [reply source/overview/product.rst]: It was primarily aesthetic - the number of languages made the TE feature list list quite long and bulky. I also wasn't certain whether I wanted to add a link to the feature comparison or use "building on the features in Team Edition, E10" and so added it there as a blanket feature. 

I'm 2/5 on it though.

Comment (jasonblais) [reply source/overview/product.rst]: Yeah, good point. My worry is if it makes the intro bulky/long with all the languages.

Comment (justinegeffen) [reply source/overview/product.rst]: I've given it another look and I agree with you; moved the languages back to TE section.

File: source/overview/product.rst [hunk 8]
```diff
@@ -24,48 +24,52 @@ Mattermost Editions
...
+- Continuous archiving
+- Multi-factor authentication
+- Highly customizable `third party bots, integrations <https://about.mattermost.com/community-applications/#publicApps>`__, and `command line tools <https://docs.mattermost.com/administration/command-line-tools.html>`__
 - Extensive integration support via `webhooks, APIs, drivers <https://docs.mattermost.com/guides/integration.html>`__ and `third party extensions <https://about.mattermost.com/default-app-directory/>`__
 - Easily scalable from dozens to hundreds of users
-- New improvements released every two months
-- Languages include U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Russian, Turkish, Spanish, and Ukrainian.
+- New features and improvements released every two months
+- Multiple languages including U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, and Ukrainian.
```
Comment (amyblais) [inline source/overview/product.rst]:
```suggestion
- Multiple languages including U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, and Ukrainian
```

**Pull request merged**